spec/models/iso_datastore_spec.rb

```diff
@@ -28,10 +28,6 @@
     end
 
     context "ems is rhv" do
-      before do
-        allow(ems).to receive(:supported_api_versions).and_return(supported_api_versions)
-      end
-
```

Comment:
Can you then also remove the let 2 lines below?

Reply:
@Fryguy yep, done.